tslint.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
@nLight Would it make sense to have this config in a similar way as our eslint config, I mean as a seperate repository? Not for now, just as a future improvement

Reply:
Isn't that the same thing as @weblancaster proposed?

Reply:
@Fabs Yeah, I did not see his comment